.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Question (Stack Overflow)
-    url: https://stackoverflow.com/questions/ask
-    about: Reach out to our Stack Overflow community.
+  - name: Questions and Chat (Slack)
+    url: http://slack.keystonejs.com/
+    about: Reach out to our Slack community with your questions, chat with the community, and get the latest announcements about Keystone Next.
```

Comment:
perhaps: 

about: Join the Keystone community Slack – a place to ask questions, share ideas, and connect to the latest announcements.

Reply:
Great, thanks!